docs/contributing/contributor-compensation.md

```diff
@@ -2,25 +2,38 @@
 
 ---
 
-## <img class="dcr-icon" src="/img/dcr-icons/RFP.svg" /> Request for Proposals (RFP)
+## <img class="dcr-icon" src="/img/dcr-icons/User.svg" /> Contractors
 
-Decred was launched with a Request For Proposal system in order to provide compensation for contributors working on larger or more significant projects. A document (RFP) would be produced which described the requirements and scope of a project, along with a clearly defined set of milestones and a reward specified in DCR. The RFP would be posted publicly and community members would be given a period of time to submit their proposals. After review, a proposal would be selected and awarded the contract.
+The term "Contractors" covers both **corporate contractors** and **independent contractors**.
 
-The RFPs were stored on GitHub and they can still be viewed. The first, [RFP-001](https://github.com/decred/RFPs/blob/master/rfp-0001/rfp-0001.md), was looking for developers to overhaul the Paymetheus application. RFP progress and payments were made public via the Decred forum[^1].
+* Corporate contractors are companies that do work for Decred, e.g. Company 0, Raedah, Ditto. Corporate contractors typically employ a number of contributors who work on Decred (but for the corporate contractor).
+* Independent contractors are people that do work for Decred. This [page](advanced/contributing-to-Decred) explains how people who contribute to the project can become independent contractors.
 
-The RFP process worked well for certain types of tasks, however a need arose to have contributors engaged and compensated over the longer term and on a more flexible basis[^2]. In January 2017 the RFP system was changed to a contractor model and Decred began to hire it's first contractors.
+Contractors contribute in a wide variety of ways including software development, design, communications, community support, research and documentation.
+ 
+Decred currently has over 50 contributors (independant contractors and employees of corporate contractors), and you can see them on [decred.org/contributors](https://decred.org/contributors/) in artfully styled portraits with badges indicating whether they work for a corporate contractor.
```

Comment:
> artfully styled portraits

This made me laugh - I think we can drop "artfully styled", it's a bit subjective.

Reply:
Is it wrong when a doc makes you laugh? 🤔